Dark Theme installation is broken

See the latest scheduled GitHub Actions run: https://github.com/eMoflon/emoflon-ibex-eclipse-build/actions/runs/12861167858

```
=> Installing plug-in: theme.
CompileCommand: exclude org/eclipse/jdt/internal/core/dom/rewrite/ASTRewriteAnalyzer.getExtendedRange bool exclude = true
Jan 20, 2025 4:50:56 AM org.apache.aries.spifly.BaseActivator log
INFO: Registered provider org.slf4j.simple.SimpleServiceProvider of service org.slf4j.spi.SLF4JServiceProvider in bundle slf4j.simple
Installing com.genuitec.eclipse.theming.core.feature.feature.group 2024.4.0.202412101111.
Installing com.genuitec.eclipse.theming.dependencies.feature.feature.group 2024.4.0.202412101111.
Installing com.genuitec.eclipse.theming.designer.feature.feature.group 2024.4.0.202412101111.
Installing com.genuitec.eclipse.theming.feature.feature.group 2024.4.0.20241210[111](https://github.com/eMoflon/emoflon-ibex-eclipse-build/actions/runs/12861167858/job/35854139479#step:8:112)1.
Installing com.genuitec.eclipse.theming.oss.feature.group 2024.4.0.202412101111.
Installation failed.
Cannot complete the install because one or more required items could not be found.
	Software being installed: DevStyle with CodeTogether Bundle 2024.4.0.202412101111 (com.genuitec.eclipse.theming.feature.feature.group 2024.4.0.202412101111)
	Missing requirement: CodeTogether Core 2025.1.0.202501171533 (com.genuitec.eclipse.codetogether.core 2025.1.0.202501171533) requires 'java.package; javax.websocket 0.0.0' but it could not be found
	Cannot satisfy dependency:
		From: CodeTogether Common API 2025.1.0.202501171533 (com.genuitec.eclipse.codetogether.api 2025.1.0.20250[117](https://github.com/eMoflon/emoflon-ibex-eclipse-build/actions/runs/12861167858/job/35854139479#step:8:118)1533)
		To: osgi.bundle; com.genuitec.eclipse.codetogether.core 0.0.0
	Cannot satisfy dependency:
		From: CodeTogether Cloud 2025.1.0.202501171533 (com.genuitec.eclipse.codetogether.feature.feature.group 2025.1.0.202501171533)
		To: org.eclipse.equinox.p2.iu; com.genuitec.eclipse.codetogether.api [2025.1.0.202501171533,2025.1.0.202501171533]
	Cannot satisfy dependency:
		From: DevStyle with CodeTogether Bundle 2024.4.0.2024[121](https://github.com/eMoflon/emoflon-ibex-eclipse-build/actions/runs/12861167858/job/35854139479#step:8:122)01111 (com.genuitec.eclipse.theming.feature.feature.group 2024.4.0.202412101111)
		To: org.eclipse.equinox.p2.iu; com.genuitec.eclipse.codetogether.feature.feature.group 1.0.0
There were errors. See log file: /home/runner/work/emoflon-ibex-eclipse-build/emoflon-ibex-eclipse-build/eclipse/configuration/1737348653398.log
Error: Process completed with exit code 13.
```

--- a/packages/theme-packages.list
+++ b/packages/theme-packages.list
@@ -1,5 +1,2 @@
 com.genuitec.eclipse.theming.core.feature.feature.group
 com.genuitec.eclipse.theming.dependencies.feature.feature.group
-com.genuitec.eclipse.theming.designer.feature.feature.group
-com.genuitec.eclipse.theming.feature.feature.group
-com.genuitec.eclipse.theming.oss.feature.group

